app/src/main/java/app/insti/Utils.java

```diff
@@ -0,0 +1,100 @@
+package app.insti;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentTransaction;
+import android.widget.ImageView;
+
+import com.google.gson.Gson;
+import com.squareup.picasso.Callback;
+import com.squareup.picasso.Picasso;
+
+import app.insti.api.RetrofitInterface;
+import app.insti.api.model.Body;
+import app.insti.api.model.Event;
+import app.insti.api.model.User;
+import app.insti.fragment.BodyFragment;
+import app.insti.fragment.EventFragment;
+import app.insti.fragment.UserFragment;
+
+public final class Utils {
+    private static String sessionId;
+    private static RetrofitInterface retrofitInterface;
+    public static UpdatableList<Event> eventCache = new UpdatableList<>();
+
+    public static final void loadImageWithPlaceholder(final ImageView imageView, final String url) {
+        Picasso.get()
+            .load(resizeImageUrl(url))
+            .into(imageView, new Callback() {
+                @Override
+                public void onSuccess() {
+                    Picasso.get()
+                            .load(url)
+                            .placeholder(imageView.getDrawable())
+                            .into(imageView);
+                }
+                @Override
+                public void onError(Exception ex) {}
+            });
+    }
+
+    public static final String resizeImageUrl(String url) {
+        return resizeImageUrl(url, 200);
+    }
+
+    public static final String resizeImageUrl(String url, Integer dim) {
+        if (url == null) { return url; }
+        return url.replace("api.insti.app/static/", "img.insti.app/static/" + dim.toString() + "/");
+    }
+
+    /** Update the open fragment */
+    public static final void updateFragment(Fragment fragment, FragmentActivity fragmentActivity) {
```

Comment:
Use `Context` as the second parameter here, and in subsequent functions. It need not always be a `FragmentActivity`

Reply:
`getSupportFragmentManager` exists only on `Fragment Activity`, so in fact it *has* to be one.

Reply:
Oh. Didn't know this. Cool.